data_catalog_backend/services/code_example_service.py

```diff
@@ -9,29 +12,39 @@
 from data_catalog_backend.schemas.User import User
 
 
+logger = logging.getLogger(__name__)
+
+
 class CodeExampleService:
     def __init__(self, session):
         self.session = session
 
     def create_code_examples(
-        self, code_examples: List[CodeExamples], user: User
+        self, code_examples_data: List[dict], resource_id: uuid.UUID, user: User
```

Comment:
Why is code_examples_data a `List[dict]`?
If you do a find usages on this function, you can see that the calls are not sending a dict, but actual objects.